src/public/modules/forms/helpers/ndjsonStream.ts

```diff
@@ -0,0 +1,71 @@
+// Modified fork of https://github.com/canjs/can-ndjson-stream to enqueue
+// the string immediately without a JSON.parse() step, as the stream payload
+// is to be decrypted by the decryption worker.
+
+// Note that this code assumes a polyfill of TextDecoder is available to run in IE11.
+
+export const ndjsonStream = (
+  response: ReadableStream<Uint8Array>,
+): ReadableStream => {
+  // For cancellation
+  let maybeReader: ReadableStreamDefaultReader<Uint8Array> | undefined
+  let shouldCancel = false
+  return new ReadableStream({
+    start: function (controller) {
+      const reader = response.getReader()
+      maybeReader = reader
+      const decoder = new TextDecoder()
+      let data_buf = ''
+
+      return reader
+        .read()
+        .then(function processResult(result): Promise<void> | undefined | void {
+          if (result.done && shouldCancel) {
+            return
+          }
+
+          if (result.done) {
+            data_buf = data_buf.trim()
+            if (data_buf.length !== 0) {
+              try {
+                controller.enqueue(data_buf)
+              } catch (e) {
+                controller.error(e)
+                return
+              }
+            }
+            return controller.close()
```

Comment:
still some differences between the old js and this file. Used to be 

```ts
controller.close()
return
```

I don't think there's an issue here, but perhaps we should stick to the old implementation since there is still a possible regression (in that the initial implementation returns `undefined` but this returns the return type of `controller.close()`